hazelcast/src/test/java/com/hazelcast/cluster/ClusterShutdownTest.java

```diff
@@ -56,11 +57,19 @@ private void testClusterShutdownWithSingleMember(final ClusterState clusterState
         final Thread thread = new Thread(new Runnable() {
             @Override
             public void run() {
-                instances[0].getCluster().shutdown();
+                while (true) {
```

Comment:
I would expect the shutdown not to fail. Also in most applications, people don't take that much care of protecting the shutdown and a whole bunch of things are shutdown sequentially. If an exception is thrown, then the following services (which could be db connection pool etc) are not closed.

So I would expect a shutdown not to fail. If it does; it must be a programming error and this should be resolved, not suppressed.